jetstream/fakespot-in-suggest.toml

```diff
@@ -199,12 +229,55 @@ select_expression = "COUNTIF(event_action = 'annoyance' and engaged_result_type
 data_source = "urlbar_events"
 [metrics.fakespot_not_relevant_dismissals.statistics.bootstrap_mean]
 
-## Review Checker
-[metrics.review_checker_displayed_count]
-select_expression = "COUNTIF(event_category = 'shopping' and event.name = 'surface_displayed')"
-data_source = "events"
-[metrics.review_checker_displayed_count.statistics.bootstrap_mean]
+
+[metrics.fakespot_4_star_A_grade]
+select_expression = "COUNTIF(mozfun.map.get_key(event_extra, 'grade') = 'A' and (CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) >= 4 and CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) < 5))"
+data_source = "fakespot_events"
+[metrics.fakespot_4_star_A_grade.statistics.bootstrap_mean]
+
+[metrics.fakespot_4_star_B_grade]
+select_expression = "COUNTIF(mozfun.map.get_key(event_extra, 'grade') = 'B' and (CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) >= 4 and CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) < 5))"
+data_source = "fakespot_events"
+[metrics.fakespot_4_star_B_grade.statistics.bootstrap_mean]
+
+[metrics.fakespot_5_star_A_grade]
+select_expression = "COUNTIF(mozfun.map.get_key(event_extra, 'grade') = 'A' and (CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) = 5))"
+data_source = "fakespot_events"
+[metrics.fakespot_5_star_A_grade.statistics.bootstrap_mean]
+
+[metrics.fakespot_5_star_B_grade]
+select_expression = "COUNTIF(mozfun.map.get_key(event_extra, 'grade') = 'B' and (CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) = 5))"
+data_source = "fakespot_events"
+[metrics.fakespot_5_star_B_grade.statistics.bootstrap_mean]
 
 ## Remote Settings Latency
 [metrics.fx_urlbar_quick_suggest_remote_settings_latency_ms]
 [metrics.fx_urlbar_quick_suggest_remote_settings_latency_ms.statistics.bootstrap_mean]
+
+[data_sources]
+
+[data_sources.fakespot_events]
+from_expression = """(
+    SELECT 
+        metrics.uuid.legacy_telemetry_client_id AS client_id,
+        DATE(submission_timestamp) AS submission_date,
```

Comment:
I think the dataset definition needs to include the experiments column. Alternatively you can use `select *` rather than specifying specific columns, see an example [here](https://github.com/mozilla/metric-hub/blob/main/definitions/firefox_desktop.toml#L2014).